content/en/agent/guide/podman-support-with-docker-integration.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Using the Docker integration with Podman container runtime
+kind: documentation
+---
+
+Podman is a daemonless container engine for developing, managing, and running OCI Containers on your Linux System. Read more on [https://podman.io/][1].
+
+Podman is an alternative to Docker as it provides a Docker-compatible CLI interface and socket. This specificity allows to use the Datadog Agent Docker integration with Podman containers.
+
+## Requirements
+
+* Podman version >= 3.2.0
+* Podman configured to expose its communication socket.
```

Comment:
More documentation about this [here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/building_running_and_managing_containers/assembly_using-the-container-tools-api_using-the-container-tools-cli) in case we want to add a link

Reply:
```suggestion
* Podman configured to expose its communication socket. [2]
```